File: CppCompiler/for loop 1.cpp
#include<iostream>
using namespace std;

int main()
{
int n,i,sum=0;
cout<<"enter range:-";
cin>>n;
for(i=1;i<=n;i++)
{
sum=sum+i;
cout<<i;
if(i<n)
{
cout<<"+";
}
}
cout<<"\nsum of  series="<<sum;
return 0;
}

File: JavaNIDE/Javaprgm/app/src/main/java/sample/Forloop1.java
package sample;
import java.util.*;

class Forloop1 
{
  public static void main (String[] ar)
  {
    int i,n,sum=0;
    System.out.println("enter range");
     Scanner m=new Scanner (System.in);
     n=m.nextInt();
     for(i=1;i<=n;i++)
     {
       System.out.print(i);
       if(i<n)
       {
         System.out.print("+");
       }
       sum=sum+i;
       
     }
     System.out.print("\nsum of n natural no="+sum);
  }
  
}
